tests/load-times/cypress/support/e2e.js

```diff
@@ -0,0 +1,3 @@
+// Ignore uncaught script errors. We can't control what happens with external
+// scripts, so...
+Cypress.on("uncaught:exception", () => false);
```

Comment:
Barely related: we should put in a test to ensure there is no logging to the browser console

Reply:
For our own code, we could have eslint throw an error for console statements (it currently throws warning, but I don't think we prevent deployments over warnings.)

For 3rd-party code (I guess USWDS and DAP?), should we also try to enforce a no-console-logging policy? I suppose it's worth doing now, and if we ever start getting log messages from some 3rd-party place, it'll force us to see it and decide what to do with it.

Reply:
Good idea. Let's stick with eslint for now, that should catch us. But if we start bringing in frameworks with dependencies (like, say, a Vue application with all the fixins), we might want the e2e version